evaluation_registry/evaluations/views.py

```diff
@@ -527,3 +533,75 @@ def evaluation_update_cost_view(request, uuid):
     evaluation = check_evaluation_and_user(request, uuid)
 
     return evaluation_cost_view(request, evaluation)
+
+
+@require_http_methods(["GET", "POST"])
+@login_required
+def evaluation_update_title_department_view(request, uuid):
+    evaluation = check_evaluation_and_user(request, uuid)
+    errors = {}
+    departments = Department.objects.all()
+
+    if request.method == "POST":
+        form = EvaluationBasicDetailsForm(request.POST, instance=evaluation)
+        form_complete = request.POST.get("form_complete")
+        selected_departments = request.POST.getlist("departments")
+        selected_lead = request.POST.get("lead_department")
+        department_to_remove = request.POST.get("remove_department")
+
+        if form_complete:
+            if form.is_valid():
+                form.save()
+
+                if form.cleaned_data["lead_department"] != evaluation.lead_department:
+                    lead_department_link = EvaluationDepartmentAssociation.objects.get(
+                        evaluation=evaluation, is_lead=True
+                    )
+                    lead_department_link.department = form.cleaned_data["lead_department"]
+                    lead_department_link.save()
+
+                to_add = set(form.cleaned_data["departments"]).difference(evaluation.other_departments) or []
```

Comment:
```suggestion
                to_add = set(form.cleaned_data["departments"]).difference(evaluation.other_departments) or set()
```
this really doesnt matter....im just surprised mypy doesnt complain here